openedx/core/djangoapps/oauth_dispatch/tests/mixins.py

```diff
@@ -14,7 +15,7 @@ class AccessTokenMixin(object):
     """ Mixin for tests dealing with OAuth 2 access tokens. """
 
     def assert_valid_jwt_access_token(self, access_token, user, scopes=None, should_be_expired=False, filters=None,
-                                      jwt_issuer=settings.DEFAULT_JWT_ISSUER, should_be_restricted=None):
+                                      should_be_asymmetric_key=False, should_be_restricted=None):
```

Comment:
Can you update the docstring Args?  I'm not sure what `should_be_asymmetric_key` is supposed to do.

Reply:
done.